File: C/program61.c
#include<stdio.h>


int SumFactor(int iNo)
{
    if(iNo < 0)
    {
        iNo = - iNo;
    }

    int iCnt = 0, iSum = 0 ;

    printf("Fcators are :\n");
    for(iCnt = 1; iCnt < iNo/2; iCnt++)
    {
        if((iNo % iCnt) == 0)
        {
          iSum = iSum + iCnt;  
        }

    }

    return iSum;
}

int main()
{
    int iValue = 0, iRet = 0;
   
    printf("Enter number :");
    scanf("%d",&iValue);

    iRet = SumFactor(iValue);

    printf("Summation o factor is %d\n",iRet);

    return 0;
}

File: Java/program61.java
import java.util.*;
 
 class Number
 {
    public  int SumFactor(int iNo)
    {
        if(iNo < 0)
        {
            iNo = -iNo;
        }
        int iCnt = 0, iSum = 0;
        for(iCnt = 1; iCnt <= iNo / 2; iCnt++)
        {
            if((iNo % iCnt) == 0)
            {
                iSum = iSum + iCnt;
            }
        }

        return iSum;
    }
 }
class program61
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iRet = 0;

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();
      


        Number eobj =new Number();

        iRet = eobj.SumFactor(iValue);

        System.out.println("Summation of factor is "+iRet);
       
    }
}